provision-virtualization-tools.sh

```diff
@@ -30,6 +30,9 @@ apt-get install -y virt-manager
 #   Could not open '/var/lib/libvirt/images/terraform_example_root.img': Permission denied'
 sed -i -E 's,#?(security_driver)\s*=.*,\1 = "none",g' /etc/libvirt/qemu.conf
 systemctl restart libvirtd
+adduser --quiet --disabled-password --gecos "vagrant" vagrant
```

Comment:
this base image that is mentioned in this repository already has this user, so please remove this hunk.

Reply:
Hi, the original script crashed on Ubuntu 21, user not exists, I assumed was missing in my host system

Reply:
Are you trying to execute this outside of the vagrant environment?

If so, you need to replace all occurrences of `vagrant` with your username. Maybe this script should be refactored to use an environment variable (e.g. `PROVISION_USERNAME` and `PROVISION_PASSWORD`?